cmd/gosubc/templates/generate_usage.txt

```diff
@@ -3,3 +3,7 @@ Usage: gosubc generate <subcommand> [arguments]
 Subcommands:
 
     version      Print version information
+
+Flags:
+-dir Directory to generate code for (default: .)
```

Comment:
Please add some white spaceing using printf style %ns formatting the n should be dynamic AND calculated based on the templates